### PR TITLE
Task 292-294: StoryCommonsense

### PR DESCRIPTION
**Pull request by Mirzyaaliii**
task292_StoryCommonsense_character_text_generation  -->  Given a short story, provide all the characters in that story.
task293_StoryCommonsense_emotion_text_generation  -->  Given a sentence, context (optional) and character; provide emotions expressed by the character in the sentence.
task294_StoryCommonsense_motiv_text_generation  -->  Given a sentence, context (optional) and character; provide motivation of the character in the sentence.

Please review.

### CODE REVIEW

**Review by Palipoor — COMMENTED** [review]
Hi,
In task 293: 
Please change `all the emotions which is ` to `all emotions which are`, or just `all emotions`.
Also, ` in the given Sentence` can be removed.
Another issue: the examples, the emotions are comma separated, which is not the case in actual instances. If the task is to list all emotions, the instances should do the same, but I think it's easier if you change the task to "one possible emotion" and change the example outputs. Otherwise, you should contain all possible permutations of, say, 4 emotions in each instance in possible outputs. Pay attention, that the outputs of instances are lists, not because the machine is going to output a list, but because each of them are possible acceptable outputs. This is also the case for task 292.

**Comment by swarooprm** [comment]
Can you address reviwer comments @Mirzyaaliii ? Overall, these are good tasks as they bring new variety to our collection.

**Comment by Mirzyaaliii** [comment]
> Hi,
> In task 293:
> Please change `all the emotions which is ` to `all emotions which are`, or just `all emotions`.
> Also, ` in the given Sentence` can be removed.
> Another issue: the examples, the emotions are comma separated, which is not the case in actual instances. If the task is to list all emotions, the instances should do the same, but I think it's easier if you change the task to "one possible emotion" and change the example outputs. Otherwise, you should contain all possible permutations of, say, 4 emotions in each instance in possible outputs. Pay attention, that the outputs of instances are lists, not because the machine is going to output a list, but because each of them are possible acceptable outputs. This is also the case for task 292.

Thanks for the comments!

- The First 2 changes are done.

- According to me, keeping only 1 emotion is not a good idea because sometimes for 1 thing we can have multiple emotions.
E.g., in the 2nd positive example,
Valerie can feel nervous as well as excited about her performance at the same time.
So, according to me, by providing all the emotions, machine can learn each and every emotion which are related to that event.

- Also, in the positive and negative examples, we need to write string only (as per the instruction) and in the instance, the output should be list. That is why emotions are comma-separated in the examples. (The same thing applies to task 292.)

File: tasks/README.md [hunk 1]
```diff
@@ -269,19 +269,24 @@ Name | Summary | Category
...
-`task287_casehold_legal_incorrect_answer_generation` | Given a prompt from a judicial decision and multiple potential holdings, choose one of the incorrect options | Legal , Incorrect Answer Generation
+`task287_casehold_legal_incorrect_answer_generation` | Given a prompt from a judicial decision and multiple potential holdings, choose one of the incorrect options | Legal, Incorrect Answer Generation
+`task288_gigaword_summarization.json` | Given a text of article, generate headline (title) for the article | Summarization
+`task289_gigaword_summarization.json` | Given the text of an article and its title, decide whether the title is appropriate for the article | Summarization
 `task290_TellMeWhy_question_answerability` | Given a short story and a question, decide whether or not the question is answerable | Classification
 `task291_semeval_2020_task4_Commonsense_Validation` | Given two statements, choose the one that makes less sense. | Answer Generation
+`task292_StoryCommonsense_character_text_generation` | Given a short story, provide all the characters in that story. | Text Generation
```
Comment (danyaljj) [inline tasks/README.md]: we gotta lowercase these names and merge with the main branch.

Comment (Mirzyaaliii) [reply tasks/README.md]: Done.

**Comment by eshaanpathak** [comment]
If you look at the "Files changed" tab, it seems like there are a lot of changes to the README outside of tasks 292 to 294. I would recommend copying the current README from `allenai:master` and just adding in the rows for tasks 292 to 294 to not make any other changes. Otherwise this would undo a lot of my edits in #224.

**Comment by danyaljj** [comment]
@Palipoor could you check if you're happy with the updated PR?

**Comment by Palipoor** [comment]
> According to me, keeping only 1 emotion is not a good idea because sometimes for 1 thing we can have multiple emotions.
Ok, so I think it's better if you change the task description to find one of the emotions(in the same manner: one of the characters) as each of the acceptable outputs in instances are one emotion/character.

**Comment by Mirzyaaliii** [comment]
> > According to me, keeping only 1 emotion is not a good idea because sometimes for 1 thing we can have multiple emotions.
> > Ok, so I think it's better if you change the task description to find one of the emotions(in the same manner: one of the characters) as each of the acceptable outputs in instances are one emotion/character.

Thank you very much for your valuable suggestion and for clarifying the problems with my tasks output.
Please review again. I made the changes.

**Comment by swarooprm** [comment]
Thanks @Mirzyaaliii. Let's wait for @Palipoor to see if she is happy or has any other suggestion.

**Comment by swarooprm** [comment]
@Palipoor, check if you have any other feedback?

**Comment by Palipoor** [comment]
Sorry I didn't see this sooner! Looks good to me!

**Review by swarooprm — APPROVED** [review]
LGTM!

**Review by swarooprm — COMMENTED** [review]
Can you fix the issues in the readme?
It's showing you are changing a lot of things which are not part of this PR. Please verify that it's not changing anything else apart from these files.
I will merge once this gets fixed.

**Comment by danyaljj** [comment]
> Can you fix the issues in the readme?
> It's showing you are changing a lot of things which are not part of this PR. Please verify that it's not changing anything else apart from these files.
> I will merge once this gets fixed.

I revised the README.

**Pull request merged**